variables.tf

```
@@ -92,6 +92,18 @@ variable "instance_count" {
   description = "The number of Tailscale Subnet Router EC2 instances you would like to deploy."
 }
 
+variable "monitoring_enabled" {
+  description = "Enable detailed monitoring of instances"
+  type = bool
+  default = true
+}
+
+variable "associate_public_ip_address" {
+  description = "Associate public IP address with subnet router"
+  type = bool
+  default = null
```

The default in the provider is false, correct? Let's be explicit about that so we don't have 3 possible types being passed for this `bool` value. 

```suggestion
  default = false
```

---

I had a brief [discussion](https://github.com/masterpointio/terraform-aws-ssm-agent/pull/18#discussion_r1474898010) with @gberenice last week on this in the SSM PR. The provider's default is that it is optional, so the null here maintains that default. However, because the original SSM agent had a hard-coded value of `false`, it would change the default behavior of this tailscale module from `false` -> `null` (optional). I can keep the original behavior of this module though if you think that's best for compatibility with existing users.